### ISSUE TITLE
Task Searching

### ISSUE THREAD

**Issue by programBreaker** [12thDoctor/kcmho#20]
When clicking on a task in the search list and clicking next, the task should be selected as the imported task.

**Comment by 12thDoctor** [comment]
I don't think we want to just import whatever task is selected, since they may not want to import **all** of the task values... 

How about having it so if the user selected a task from the search results but did ***not*** import it and click on the Next button, then we show a prompt:

> Do you want to import the selected task?
> [Yes, import all task details] [Yes,
